packages/create-skill-app/templates/next/src/pages/index.tsx

```diff
@@ -1,12 +1,28 @@
 import Layout from 'components/layout'
 import type {NextPage} from 'next'
+import {trpc} from '../trpc/trpc.client'
+import {createAppAbility} from '@skillrecordings/skill-lesson/utils/ability'
+
+const useAbilities = () => {
+  const {data: abilityRules} = trpc.abilities.getAbilities.useQuery()
+
+  return createAppAbility(abilityRules || [])
+}
```

Comment:
the changes in this file are meant to demonstrate how to start using the `tRPC` queries in app code